SingularityService/src/main/java/com/hubspot/singularity/hooks/LoadBalancerClientImpl.java

```diff
@@ -68,6 +72,54 @@ public LoadBalancerClientImpl(SingularityConfiguration configuration, ObjectMapp
     this.mesosProtosUtils = mesosProtosUtils;
   }
 
+  public String getLoadBalancerUri() {
```

Comment:
are these getters used anywhere? No need to make these public when we can just use the private variables

Reply:
We use the getters in the SingularityUpstreamChecker, particularly in the taskToUpstreamInfo function